### ISSUE TITLE
Announcement: @haikuginger is the lead maintainer until further notice

### ISSUE THREAD

**Issue by shazow** [shazow/urllib3#1290]
Hi everyone who watches urllib3 issues. 👋 

As you may recall from #909, @Lukasa has been the lead maintainer of urllib3 for the past year and a half. I am very grateful for Cory's commitment of time and effort, he has done an excellent job pushing urllib3 forward. As employment arrangements change and life marches on, Cory informed me that it's time for a new lead maintainer to take the torch.

## New maintainer

@haikuginger will be taking responsibility as the lead maintainer and thus the direction of urllib3 moving forward.

As before, our family of contributors and secondary maintainers will continue to support our new leader as needed.

## Logistics

This is our second transition, so let's make a checklist of everything that needs to happen for this to be officially complete:

- [x] Announcement
- [ ] Github admin
- [ ] PyPI admin
- [ ] ReadTheDocs admin

Anything else? (Maintainers, please edit the checklist above and check it off as it's completed.)

---

If anyone has questions or concerns, comments here are welcome or email me. ❤️

**Comment by Lukasa** [comment]
Very excited to embrace our new world with @haikuginger running the show!

**Comment by haikuginger** [comment]
Okay, first thing we do is throw out all the existing code and start over...

Kidding. I'm very excited to be taking over the leadership of urllib3, and look forward to working with the existing team to get where we're going (wherever that is).

First things first, I'll be going through existing code and issues to make sure my 10,000-foot view of the project is fully up-to-date. Once I've gotten that done, we can figure out what we need to accomplish as part of the process to get to urllib3 2.0.

**Comment by haikuginger** [comment]
@shazow, I'm not sure if Github admin has been transferred correctly - I don't have access to the Admin tab for this repo that I do for my own.

**Comment by shazow** [comment]
@haikuginger You're a collaborator on the repo. Is there something more aside from a permanent ownership transfer? Looking around, don't see what there is to do. We can also transfer urllib3 to an org which I think would allow for better permission handling if anything is missing.

**Comment by Lukasa** [comment]
Yeah, we discussed this when I took over: repos under user namespaces cannot have full admin privileges for anyone other than the person whose namespace they’re in.

**Comment by shazow** [comment]
Anyone know if it's possible to add team members to public projects on AppVeyor and TravisCI?

**Comment by haikuginger** [comment]
I've added "release process documentation" to the list of TODOs; based on a quick read, it seems like the process would be to checkout the `release` branch, then update the version at the top of the `CHANGES` file, but confirmation on that would be... good.

**Comment by shazow** [comment]
@haikuginger Good idea. Once upon a time, I wrote [release.sh](https://github.com/shazow/urllib3/blob/master/release.sh) which handles the release process. I'm not sure if @Lukasa still uses it?

**Comment by Lukasa** [comment]
I do still use it!
